### DISCUSSION TITLE
📢 Welcome to magento-lts Discussions!

### DISCUSSION THREAD

**Discussion by Flyingmana** [OpenMage/magento-lts#1351] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community, and what you would like to see here in the future.

**Comment by Danieleeffe1** [comment]
Good morning to the Magento LTS community

I was very happy to find that this community exists

I apologize for my English, I use google translate.

First of all, let me introduce myself:
I'm Daniele and we have been using magento since 2014.

I am not a programmer, but a web designer.

My knowledge of php javascript code is nil and I have always limited myself to "copy / paste".

Not having the basics, I have difficulty understanding how to switch from magento classic to OpenMage LTS.

I don't understand composer's dynamics, I tried to gain experience but as I said before I'm a "copy / paste".
I'm a stupid kid who needs to see all the individual steps.

I installed an OpenMage LTS 20.0.4 and noticed that you have eliminated the compilation.

I read that you did it because it was considered obsolete, however I ask if I can reactivate it in my installation.

My request comes from the comparison between OpenMage and Magento classic in my current Hosting, classic Magento after compilation is more responsive. I tested with google page speed, GtMatrix and the WebViewTest android app (I highly recommend it for all those who wish to have exact times on the loading of web pages on mobile devices).

Maybe there is some option to activate that I didn't understand?

sooner or later I will have to switch from the current Magento 1.9.4.5 to your OpenMage but my current hosting does not provide a CLI and not having the basics I have difficulty in implementing and understanding how to switch from magento classic to OpenMage LTS without invalidating all my work and plug-ins installed.

My limited knowledge of English didn't help me.

Everything I learned about magento I discovered with many difficulties looking for solutions to the various problems that arose over time, however, I obtained the results I would like to share.

I modified the css and some phtml files of the RWD template to get a responsive template closer to mobile.

In particular:
Better use of space on small mobile devices.
Modification of the phtml Header to always have the menu available.
Edit the phtml List page and product page to have a Label attribute indicating a promotion.
In Bundle products Adding thumbnails on child products for CheckBox and Buttons options.
Added code for loading Iframe Video in image gallery product from attribute text area (to be improved)
In the TextArea Product Quantity change the tag Type = "textarea" to inputmode = "numeric" to get the numeric keypad with mobile devices.
Added Apple Icon Link to Increase SeO Rating.
Added Meta tag to prevent finger zooming.

An example here (magento 1.9.4.5 classic):
https://test.monsterbox.it

I have installed several plug-ins downloaded from Githhub and there are a couple that I would like to propose to integrate natively into Open Mage.

These plug-ins can enhance the OpenMage browsing experience.

I have written a lot.
Many concepts are poorly translated.
I apologize.
I just want to help OpenMage grow

**Reply by kiatng** [reply]
What PHP version are you using? Do you have redis cache?

**Comment by Danieleeffe1** [comment]
php7.2
No, I don't have redis cache and I don't have the ability to install
packages, I don't have server control.

My hosting is a Virtual hosting with pre-installed packages I don't have a
terminal and permissions to install software packages

Il mer 16 dic 2020, 14:25 Daniele Effe <daniele.effe1@gmail.com> ha scritto:

**Comment by LuizSantos1** [comment]
Hi there,
I don't know where to suggest this
I used an extension for tracking codes which was really good and complete. 
It had several carriers and tracking status was constantly updated (I guess they used a script on their site)

Unfortunatelly, they closed the company, they looked like "Aftership", but not quite the same, as they have 
created all graphical tracking design inside Magento, so all the tracking experience was inside our website, which
was really greate. 

The company was called "Packpin" and I still have the extension I can adapt it but unfortunatelly real time tracking
status does not work anymore.  

So I wonder if a developer could adapt it and insert a script to make this feature work again. 
I have scanned the plugin at virustotal.com to make sure it is clean

That would be awesome if it could be adapted and used by the community, maybe even
be OpenMage's default "featured extension" in the future? 
[pp tracking.zip](https://github.com/OpenMage/magento-lts/files/6058228/pp.tracking.zip)

Regards
Luiz Santos

**Comment by LuizSantos1** [comment]
Hi again, and here goes another suggestion for search engine
Magento Default search is not relevant enough, there is an Open Source 
extension called "Bubble Autocomplete",it was created by Johann Reinké. 
Here is the orignal link: https://github.com/jreinke/magento-autocomplete

But the original one has some issues that was not fixed by the creator. 

So by the time I have made some modifications to make it work with my theme (Fortis) 
As it was too specific, so when I changed the theme, I have left this extension behind. 

But looking at it again, I have found a developer called Frederik Winkelsdorf which 
has improved the original module and fixed some issues. 
Here is the link for the improved forked version I am using right now: 
https://github.com/winkelsdorf/magento-autocomplete

Yes, I know the plugin is quite old, but I have made just a few css
arrangements in my website and it is working like a charm. 

In my opinion, this is one if not the best Open Source search autocomplete for Magento/OpenMage

I have tested and I am using it right now with latest OpenMage Magento LTS 1.9.4.11
using PHP 7.3 (it worked with PHP 7.4 was well) and even with my website being optmized
by LightSpeed, it still works. 

I hope it helps someone even if it won't be considered by the OpenMage team! ;)

Best regards!

**Comment by Danieleeffe1** [comment]
Good morning

thanks for the reply and for the advice.



Il giorno mar 2 mar 2021 alle ore 20:15 LuizSantos1 <
notifications@github.com> ha scritto:
